test/Chainweb/Test/Pact/Checkpointer.hs

```diff
@@ -418,6 +432,39 @@ begin pactdb = _beginTx pactdb Transactional
 commit :: PactDb e -> Method e [TxLog Value]
 commit pactdb = _commitTx pactdb
 
+readRowUnitTest :: Assertion
+readRowUnitTest = simpleBlockEnvInit runUnitTest
+  where
+    writeRow' pactdb writeType conn i  =
+      _writeRow pactdb writeType (UserTables "user1") "key1"
+      (ObjectMap $ M.fromList [("f", (PLiteral (LInteger i)))]) conn
+    runUnitTest pactdb e schemaInit = do
+      conn <- newMVar e
+      void $ schemaInit conn
+      let user1 = "user1"
+          usert = UserTables user1
+      void $ begin pactdb conn
+      _createUserTable pactdb user1 "someModule" conn
+      writeRow' pactdb Insert conn 1
+      void $ commit pactdb conn
+      let numWrites = 100 :: Int
+          loop current =
+            if current == numWrites
+              then return ()
+              else do
+                void $ begin pactdb conn
+                writeRow' pactdb Update conn $
+                   fromIntegral $ current + 1
+                void $ commit pactdb conn
+                loop (current + 1)
+      loop 1
+      r <- _readRow pactdb usert "key1" conn
+      case r of
+        Nothing -> assertFailure "Unsuccessful write"
+        Just (ObjectMap m) -> case M.lookup "f" m of
+          Just l -> assertEquals "lol" l (PLiteral (LInteger 100))
```

Comment:
"lol"?

Reply:
I'll fix that. I had intended to go back to that.